src/modules/function.js

```diff
@@ -28,9 +28,20 @@ export default function addTrash(event) {
   newBtn.innerHTML = '<i class="fa fa-trash-alt"></i>';
   parentElement.append(newBtn);
   liItem.style.backgroundColor = '#ffe6ee';
-  const textElem = liItem.querySelector('p');
-  textElem.setAttribute('contenteditable', true);
-  textElem.focus();
+  const editTask = liItem.querySelector('p');
+  editTask.setAttribute('contenteditable', true);
+  // editTask.focus();
```

Comment:
Your app is working as expected. But in this project, we are not allowed to use unused commented codes! Please remove this line :)